tests/etl/test_loaders.py

```
@@ -0,0 +1,51 @@
+
+def test_postgresql_loader_load_appends_rows():
+    db = FakeDBManager()
+    loader = PostgreSQLLoader(db)
+    df = pd.DataFrame({"a": [1, 2]})
+    rows = loader.load(df, "tbl")
+    assert rows == 2
+    assert "tbl" in db.tables
+    pd.testing.assert_frame_equal(db.tables["tbl"].reset_index(drop=True), df)
+
+
+def test_table_mapping_loader_routes_data():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_table_mapping_loader_routes_data`

Add return type annotation: `None`

(ANN201)